### DISCUSSION TITLE
Finding a ReaLearn preset for Arturia Minilab II?

### DISCUSSION THREAD

**Discussion by jonnners** [helgoboss/helgobox#1292] [Q&A]
Found it under 'User Presets'. Can't find a way to delete this message though.

Amazing work, and I'm super impressed with the amount of support work you've done.

Am I right in believing there is a ReaLearn preset for Arturia Minilab II? 

I have installed the Minilab II package via ReaPack but cannot see any option for it in the Presets drop down.

Do I need to find the file .json and add it manually? 

I am on Linux Ubuntu if that makes a difference.

**Comment by helgoboss** [comment]
There is a **controller preset** (in the controller compartment) for the mk2 available on ReaPack. It will appear after installing it, switching to the controller compartment and pressing "Menu => Compartment presets => Reload all compartment presets from disk". I will slowly transition to make all of those ReaPack presets available as built-in factory presets in the next releases, so you will probably find it there in the future.
